pkg/cmd/serviceaccount/create.go

```diff
@@ -50,7 +51,7 @@ func newCreateCmd(authProvider auth.Provider) *cobra.Command {
 	createRunner.AppendPhases(
 		phases.NewAADApplicationPhase(),
 		phases.NewServiceAccountPhase(),
-		phases.NewFederatedIdentityPhase(),
+		phases.NewFederatedIdentityPhase(uuid.New().String),
```

Comment:
The only drawback with using an `uuid` for every request is we can end up creating a fic for same <namespace, name, issuer> when the command is run multiple times with different uid as name. Should we generated a name deterministically based on `<namespace, name, issuer>` tuple that's expected to be unique for an app?

Reply:
That's fair. Updated the PR.